etl/transformer/mainnet_impl.go

```diff
@@ -69,7 +70,7 @@ func (m *MainNetTransformer) run(ctx context.Context) {
 			return
 		}
 		go func() {
-			belogger.FromContext(ctx).Infof("transformed jet drop to canonical: %v", transform)
+			go log(ctx, transform)
```

Comment:
why go ?

Reply:
I don't want to suspend data transferring. 
theoretically, the log might take some time. and be a part of the bottleneck. that's why I prefer to log in goroutine

Reply:
if use `go`, we can't see in log in what time (or before/after what operations) this message was really produce. May be for now don't do such complex things with data, just log pulse_number and jet_id, as at `process` component. If we ever feel need for more information here, we can add it later.
We can also call log inside `for` , so we can get every jetDrop data

Reply:
I think we really can afford print all data without `go`
I suggest to log without goroutine deleting `go`. WDYT?